crates/compiler/test_syntax/tests/snapshots/pass/minus_twelve_minus_five.expr.formatted.roc

```diff
@@ -1 +1 @@
--12 - 5
\ No newline at end of file
+-12 -5
```

Comment:
This doesn't look correct. We are changing the semantic meaning of this expression.